common/src/main/java/org/apache/celeborn/common/meta/MapFileMeta.java

```diff
@@ -17,16 +17,38 @@
 
 package org.apache.celeborn.common.meta;
 
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
 public class MapFileMeta implements FileMeta {
   private int bufferSize;
   private int numSubPartitions;
   private String mountPoint;
+  private Map<Integer, Integer> subPartitionWritingSegmentId;
+  private List<SegmentIndex> subPartitionSegmentIndexes;
+  private volatile boolean isWriterClosed;
```

Comment:
`writer always closed as this is committed file info.`, when is the `isWriterClosed` false?